ls_0/index.html

```diff
@@ -0,0 +1,380 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/ls_0/style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="container">
+
+        <header>
+            <nav>
+                <div class="nav-main">
+                    <div class="nav-main-left"><img src="/ls_0/src/images/Union.svg" alt="logo"></div>
+                    <div class="nav-main-right">Explore</div>
+                    <div class="nav-menu">
+                        <a href="#" class="nav-list">Сategories</a>
```

Comment:
If we a talking about class names. This not looks suitable. _nav-list_ implies that this exactly the list of some items. So class name like this more suitable for parent element which includes some nav items. Here is better, for example, _nav-item_